cloudformation/cloudformation.yaml

```diff
@@ -121,7 +121,7 @@ Resources:
                       Statement:
                           - Effect: Allow
                             Action: s3:GetObject
-                            Resource: !Sub arn:aws:s3:::fulfilment-private/${Stage}/*
+                            Resource: !Sub arn:aws:s3:::gu-reader-revenue-private/membership/fulfilment-lambdas/${Stage}/*
```

Comment:
I don't think we need to keep the permissions for the old bucket since the step functions and the comparator are both called on a schedule so we can be sure that they will not execute in the middle of the deploy